scripts/externalize_js.sh

```diff
@@ -0,0 +1,75 @@
+#!/bin/sh
+
```

Comment:
Might want to add `set -e` to the top here so it will exit on any failures.  Maybe `set -euo pipefail` as used [here](https://github.com/dimagi/commcare-hq/blob/master/scripts/update-translations.sh#L2-L2)

Reply:
Oooh, that's cool. Adding it